lib/msal-core/src/telemetry/TelemetryTypes.ts

```diff
@@ -13,10 +13,16 @@ export interface TelemetryPlatform {
     sdkVersion: string;
     applicationName: string;
     applicationVersion: string;
+    networkInformation?: NetworkInformation
 }
 
 export interface TelemetryConfig {
     platform: TelemetryPlatform;
     onlySendFailureTelemetry?: boolean;
     clientId: string;
 }
+
+// In the browser this commes from navigator.connection
+export interface NetworkInformation {
+    effectiveType: string;
```

Comment:
Can this be an enum, since there is a known set of expected value?

Reply:
I would prefer not, since we don't do anything with the value programatically. we just want to dump whatever the browser holds here